exercises/practice/hello-world/.docs/instructions.append.md

```diff
@@ -40,98 +40,16 @@ in order to solve the current problem.
 # Solving "Hello, World!"
 
 Before proceeding any further, make sure you have completed the required setup
-steps described by the links below:
+steps described by the link below:
 * [Installing Java and Gradle](https://exercism.org/docs/tracks/java/installation);
```

Comment:
This link is only needed if solving locally, but currently the description gives the idea that you **must** do this. we can make this distinction about local vs online editor here. We can say something along the lines "you can use our online editor to solve your solution and run the tests, but if you want to solve and run the tests locally, check these links". 

A link to [working locally](https://exercism.org/docs/using/solving-exercises/working-locally) also can be useful here.